lib/content-services/src/lib/search/components/search-filter-chips/search-filter-chips.component.ts

```diff
@@ -36,4 +36,7 @@ export class SearchFilterChipsComponent {
         public queryBuilder: SearchQueryBuilderService,
         public facetFiltersService: SearchFacetFiltersService) {}
 
+    get facetChipTabbedId() {
```

Comment:
@MichalKinas this kind of getters are also disallowed. If you call function in getter and then getter is called during change detection in html then that function will be called too like it would be directly in HTML (you use join function here). 

We can use only simple getters without extra functions and too complex logic. Can we avoid that somehow?

Reply:
I used different approach setting the id when search results gets updated, please check if that works for you

Reply:
thanks, now it looks correct :)